extensions/manual-groups.py

```diff
@@ -398,21 +413,21 @@ async def studyCourseCount(self, ctx, role: typing.Optional[str]):
                 all_count += 1
```

Comment:
Should external students be recognized for stats?

Reply:
As this is only a temporary solution, I did not care at the time